HISTORY.rst

```diff
@@ -9,6 +9,8 @@ History
 
 * ``metro_code`` on ``geoip2.record.Location`` has been deprecated. The
   code values are no longer being maintained.
+* The type hinting for the optional `locales` keyword argument now allows
```

Comment:
Should there be two `s?

Reply:
Yes! We need an reStructuredText linter as I am always trying to write Markdown.